Update faq.md (#4343)
SumoLogic/sumologic-documentation: docs/manage/partitions/flex/faq.md

## How can I optimize Flex data search costs?

Below are some ways to control the cost of running queries data in Flex:

* Logically apportion your data, depending on how you plan to query it, into multiple partitions, instead of creating a single large partition. Try to limit queries to a single partition as much as possible. Assume you have three different services sending logs to Sumo Logic. There a couple of approaches you can take:
    * **Create one partition per service**. All the logs for that service go there. This approach can be leveraged if each team owns a service, and they typically query the logs within their service to find issues.
    * **Create one partition per log type (debug, infra, and so on)**. This approach can be leveraged if, most of the time, the team searches one log type per query. For example, the team only queries debug logs, and typically queries other logs like Database logs and OS logs in separate queries. 
* Try to use a restrictive time range, instead of running a query with a very long time range.

> * Use indexes with the metadata fields to narrow down the volume of data scanned. This in turn will help you to reduce the scan cost.

Author: ```suggestion
* Use indexes with the metadata fields to narrow down the volume of data scanned. This in turn will help you to reduce the scan cost. For example, if you're querying data from the `k8s_host` source and it's being sent to multiple partitions, you can limit the scope of your search to the `abc` partition by using `_index=abc` for the selected time range.
    ```sql
    _index=abc AND _sourceCategory=k8s_host
    ```
```

> * Using index/indexes with the metadata fields will narrow down the volume of data scanned, this in turn will help you to reduce the scan cost.

Reviewer: ```suggestion
* Use indexes with the metadata fields to narrow down the volume of data scanned. This in turn will help you to reduce the scan cost.
```

> * Using index/indexes with the metadata fields will narrow down the volume of data scanned, this in turn wil help you to reduce the scan cost.

Author: ```suggestion
* Using index/indexes with the metadata fields will narrow down the volume of data scanned, this in turn will help you to reduce the scan cost.
```
